### ISSUE TITLE
Address query to PublicAPIService::GetObject throws error

### ISSUE THREAD

**Issue by jfriv** [theQRL/QRL#1027]
Requests to PublicAPIService::GetObject with a query that is an valid QRL Address result in the following error:
```
2018-03-10 22:24:44,290|0.61.3|synced| ERROR : Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/qrl/services/grpcHelper.py", line 20, in wrap_f
    return f(caller_self, request, context)
  File "/usr/local/lib/python3.5/dist-packages/qrl/services/PublicAPIService.py", line 162, in GetObject
    answer.address_state.CopyFrom(address_state)
TypeError: Parameter to CopyFrom() must be instance of same class: expected qrl.AddressState got AddressState.
```

**Comment by som-dev** [comment]
I tried to reproduce this in v0.62 but could not do so.  Do you still see this issue?  Was there a particular client language you were using to access the API?  Or maybe a specific address that caused the issue?

**Comment by cyyber** [comment]
Issue was fixed on PR #1037

**Comment by jfriv** [comment]
Thanks @cyyber !  If I was more experienced with python I would have attempted a fix myself
